IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometryResource/Entities/IfcGradientCurve/Documentation.md

```diff
@@ -1,3 +1,5 @@
 Gradient curve is a type of curve 3D curve representation that is based on its 2D projection (BaseCurve) and a height deifned by its gradient segments which can be derived from a function that retrieves it from the segment start height, its placement and the ParentCurve instance and the type of the ParentCurve.
```

Comment:
```suggestion
Gradient curve is a type of 3D curve representation that is based on its 2D projection (BaseCurve) and a height function (IfcGradient). The curve is defined on the curvilinear surface, with its first (horizontal) axis being the BaseCurve and the second (vertical) axis being perpendicular to the plane in which the BaseCurve lies. 
```

I would put the sentence about the gradient function into the documentation of the `IfcGradient` function.